taxcalc/records.py

```diff
@@ -713,7 +719,7 @@ def _impute_variables(self):
                                                              self.MARS == 6)),
                                  2., 1.)
         # impute number of extra standard deductions for aged
-        self._numextra = np.where(np.logical_and(self.FDED == 2, self.e04470 <
+        self._numextra = np.where(np.logical_and(self.FDED == 2, self.e04470 >
```

Comment:
This is a typo: if `e04470 < std_2009`, then `e04470 - std_2009` would be negative.